libnmstate/validator.py

```diff
@@ -95,10 +95,24 @@ def validate_interface_capabilities(ifaces_state, capabilities):
 
 
 def validate_interfaces_state(original_desired_state, current_state):
+    _validate_new_interface_state(original_desired_state, current_state)
     validate_link_aggregation_state(original_desired_state, current_state)
     _validate_linux_bond(original_desired_state, current_state)
 
 
+def _validate_new_interface_state(original_desired_state, current_state):
```

Comment:
How about `_validate_link_administrative_state` ?

In this context, mentioning `original_desired_state` is confusing, I think just a simple `desired_state` is enough.